Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLReduction2.py

```diff
@@ -229,6 +230,45 @@ def PyInit(self):
 
         self.setPropertySettings('OutputFluxWorkspace', beam)
 
+    def _add_correction_information(self, ws):
+        """Adds information regarding corrections and inputs to the provided workspace.
+
+        Args:
+            ws: (str) workspace name to which information is to be added
+        """
+        mtd[ws].getRun().addProperty(
+            'numor_list',
+            common.return_numors_from_path(self.getPropertyValue('Runs')),
+            True)
+        mtd[ws].getRun().addProperty(
+            'sample_transmission_ws',
+            common.return_numors_from_path(self.getPropertyValue('TransmissionWorkspace')),
+            True)
+        mtd[ws].getRun().addProperty(
+            'container_ws',
+            common.return_numors_from_path(self.getPropertyValue('EmptyContainerWorkspace')),
+            True)
+        mtd[ws].getRun().addProperty(
+            'absorber_ws',
+            common.return_numors_from_path(self.getPropertyValue('DarkCurrentWorkspace')),
+            True)
+        mtd[ws].getRun().addProperty(
+            'beam_ws',
+            common.return_numors_from_path(self.getPropertyValue('EmptyBeamWorkspace')),
+            True)
+        mtd[ws].getRun().addProperty(
+            'flux_ws',
+            common.return_numors_from_path(self.getPropertyValue('FluxWorkspace')),
+            True)
+        mtd[ws].getRun().addProperty(
+            'sensitivity_ws',
+            common.return_numors_from_path(self.getPropertyValue('SensitivityWorkspace')),
+            True)
+        mtd[ws].getRun().addProperty(
+            'mask_ws',
+            common.return_numors_from_path(self.getPropertyValue('MaskWorkspace')),
+            True)
+
```

Comment:
Could this function and the similar one in SANSILLReduction be moved to SANSILLCommon instead of being duplicated? Since it now also contains `add_correction_numors`, that seems more coherent and clearer

Reply:
Thanks for the change!